spec.bs

```diff
@@ -224,21 +224,21 @@ dictionary URLPatternComponentResult {
 };
 </xmp>
 
-Each {{URLPattern}} object has an associated <dfn for=URLPattern>protocol component</dfn>, a [=component=], which must be set upon creation.
+Each [=URL Pattern=] has an associated <dfn for=URLPattern>protocol component</dfn>, a [=component=], which must be set upon creation.
```

Comment:
Since these are not properties of the class, they should not live here. Also, their `for=""` values should change, from `for=URLPattern` (refers to the class) to `for="URL pattern"` (refers to the struct).

Reply:
done.